tests/hlsl/dxsdk/FluidCS11/FluidRender.hlsl.expected

```diff
@@ -0,0 +1,220 @@
+result code = 0
```

Comment:
Unfortunately, checking in DXBC assembly baselines like this doesn't usually work, for the simple reason that different versions of the fxc compiler produce different output (and not just a different `Generated by ...` line - actual code differences too).

The usual approach in our tests is to have these tests where we run fxc and slangc on the same input file and then diff the output; in those cases we need to do a bit of `#ifdef` magic to make sure that fxc sees names with the `_0` suffix while Slang sees them without.